micrometer-core/src/main/java/io/micrometer/core/instrument/binder/kafka/KafkaMetrics.java

```diff
@@ -87,20 +88,34 @@
     }
 
     @Override public void bindTo(MeterRegistry registry) {
+        commonTags = getCommonTags(registry);
+        prepareToBindMetrics(registry);
+        checkAndBindMetrics(registry);
```

Comment:
I should have changed this in a separate commit, but note I've made sure we're calling `checkAndBindMetrics` synchronously on binding to ensure that has been done once by the return of the binding. This may allow us to simplify some of the test code that otherwise explicitly calls it.